### PR TITLE
Added the spin projection operators

### PR DESCRIPTION
**Pull request by AngelDPS**

### CODE REVIEW

File: utilities/wannier2sparse/include/operator_utils.hpp [hunk 1]
```diff
@@ -35,7 +35,8 @@ namespace oputil
 	string AVAIL_OPS[] = {	
 							"VX"  ,"VY"  ,"VZ","VXSX","VXSY","VXSZ",
 							"VYSX","VYSY","VYSZ","VZSX","VZSY","VZSZ",
-							"SX", "SY", "SZ","TX","TY", "TZ"	
+							"SX", "SY", "SZ", "PUSX", "PUSY", "PUSZ",
+							"PDSX", "PDSY", "PDSZ", "TX","TY", "TZ"
```
Comment (AngelDPS) [inline utilities/wannier2sparse/include/operator_utils.hpp:39]: Pushed the torque operators back to keep the projection operators next to the spin ones. Decided on PxSy where _x_ can be U for **U**P and D for **D**own and _y_ the usual X, Y, Z spin directions. Due to the unfortunate coincidence of naming the projection operator for the up, y spin _PUSY_ , I'm open to changes in the naming scheme

File: utilities/wannier2sparse/include/operator_utils.hpp [hunk 2]
```diff
@@ -128,6 +141,18 @@ namespace oputil
...
+	 *
+	 * @param[out] direction  The direction of the projection which can be = +1,-1;
+	 * @param[in]  op string containing the operator label.
+	 */
+	 inline
+	 char projection_direction(string op)
+	 {
+		 return op[1];
+	 };
+
```
Comment (AngelDPS) [inline utilities/wannier2sparse/include/operator_utils.hpp:155]: This whole function could be scraped in favour of using the velocity_direction function which does the same,

File: utilities/wannier2sparse/include/operator_utils.hpp (hunk 1 above)
Comment (adamecius) [reply utilities/wannier2sparse/include/operator_utils.hpp:39]: I think this naming convention is fine, sometimes is good to have a laugh.

File: utilities/wannier2sparse/include/operator_utils.hpp (hunk 2 above)
Comment (adamecius) [reply utilities/wannier2sparse/include/operator_utils.hpp:155]: Let's keep it for the moment. Once the code is finished, we can developed some testing units and then can safely change things.

**Review by adamecius — APPROVED** [review]
Included spin projection operator in wannier2sparse module

**Pull request merged**